src/dkg/participant.rs

```diff
@@ -249,7 +250,7 @@ impl<C: CipherSuite> Participant<C> {
             &mut rng,
         )?;
 
-        // Unwrapping cannot panic here
+        // This unwrap() cannot fail, as we always have at least an empty vector of coefficients.
         let coefficients = coeff_option.unwrap();
```

Comment:
Note that although I haven't changed the type of `coefficients` here, it may be worth it to consider having `Participant::new_internal` output directly a vector of coefficients. The rationale behind having it as an option was originally to differentiate between signers and dealers, but this method is private and coefficients are not exposed in the signer case, hence we could safely output a `Vec` no matter the distinction. This would be a tad cleaner in my opinion.